tools/Vrops.py

```diff
@@ -248,13 +248,19 @@ def get_latest_stats_multiple(self, target: str, token: str, uuids: list, keys:
     def _get_chunk(q, uuid_list, url, headers, keys, target, kind, collector, chunk_iteration):
         logger.debug(f'chunk: {chunk_iteration}')
 
+        # Indicates whether to report only "current" stat values, i.e. skip the stat-s that haven't published any value
+        # during recent collection cycles.
+        current_only = True
```

Comment:
Are you planning to make this configureable?

Reply:
Yes